### ISSUE TITLE
Fails to save after rotating some PDFs

### ISSUE THREAD

**Issue by acheronfail** [jeromerobert/pdfarranger#110]
I'm still seeing this issue:

Running Ubuntu 19.04, and using version `1.3.0` of `pdfarranger`

```
Traceback (most recent call last):
  File "/home/acheronfail/.local/lib/python3.7/site-packages/pdfarranger/pdfarranger.py", line 593, in choose_export_pdf_name
    self.save(only_selected, file_out)
  File "/home/acheronfail/.local/lib/python3.7/site-packages/pdfarranger/pdfarranger.py", line 196, in wrapper
    func(*args, **kwargs)
  File "/home/acheronfail/.local/lib/python3.7/site-packages/pdfarranger/pdfarranger.py", line 634, in save
    exporter.export(self.pdfqueue, to_export, file_out)
  File "/home/acheronfail/.local/lib/python3.7/site-packages/pdfarranger/exporter.py", line 98, in _pypdf2
    current_page.rotateClockwise(angle)
  File "/usr/lib/python3/dist-packages/PyPDF2/pdf.py", line 2150, in rotateClockwise
    self._rotate(angle)
  File "/usr/lib/python3/dist-packages/PyPDF2/pdf.py", line 2166, in _rotate
    self[NameObject("/Rotate")] = NumberObject(currentAngle + angle)
TypeError: unsupported operand type(s) for +: 'IndirectObject' and 'int'
```
I installed from source via:

```
pip3 install --user --upgrade -r https://raw.githubusercontent.com/jeromerobert/pdfarranger/master/requirements.txt
```

Here's a PDF which produces the error. I try to rotate the first 5 pages and save, but the error prevents me from doing so.

[11092019104002-0001.pdf](https://github.com/jeromerobert/pdfarranger/files/3607467/11092019104002-0001.pdf)

**Comment by acheronfail** [comment]
@jeromerobert I just tried again with your patch (without `pikepdf`) and I get the following error:

```
name 'generic' is not defined
```

This is using the same PDF file.

**Comment by jeromerobert** [comment]
Ouch ! Sorry and thank you. Fixed in 33979dd

**Comment by acheronfail** [comment]
Never apologise for open source software! You've already given all of this to us freely. :laughing: 

Confirmed it's working now - appreciate the quick fixes!
